### ISSUE TITLE
Handling Multiple Minecraft Accounts via one profile field

### ISSUE THREAD

**Issue by bl968** [DawnFire-Realms/CommunityBridge#213]
I would like a delimiter that would be used to identify multiple accounts through one profile field. 

IE 
minecraftaccount1,minecraftaccount2 
minecraftaccount1:minecraftaccount2 

If present Community bridge would consider any of the listed accounts to be a valid member. 
I have 2 minecraft accounts and use both but do not want to have to create two forum accounts.

**Comment by TEKAA92** [comment]
Good idea, I will need to do some tests to see how to best solve this.
Sorry for the late response.
